pom.xml

```diff
@@ -1907,6 +1911,26 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                      <groupId>org.owasp</groupId>
+                      <artifactId>dependency-check-maven</artifactId>
+                      <version>${dependency-check-maven.version}</version>
+                      <configuration>
+                          <cveValidForHours>24</cveValidForHours>
+                          <failBuildOnCVSS>${dependency-check-maven.failBuildOnCVSS}</failBuildOnCVSS>
+                          <skipProvidedScope>true</skipProvidedScope>
+                          <skipSystemScope>true</skipSystemScope>  <!-- avoid error when processing jdk.tools:jdk.tools:jar:1.8:system -->
+                          <suppressionFile>owasp-dependency-check-suppressions.xml</suppressionFile>
+                      </configuration>
```

Comment:
This can be omitted. It looks like the plugin will pick up the configuration of the `dependency-check-maven` plugin described above so we don't need to duplicate this in the pom